next.config.js

```diff
@@ -1,8 +1,12 @@
+const execSync = require('child_process').execSync
+
 module.exports = {
   typescript: {
     ignoreDevErrors: true,
   },
   webpack: (config) => {
+    execSync('npm run update-supported-browsers')
```

Comment:
Nice so it works without checking for `isSever`. Just run it locally and was fine. 

Should we also add `lib/supportedBrowsers.js` to `.gitignore`?

Reply:
Or maybe put it under `lib/generated` and `.gitignore` that. I've also have some types on the `i18n` branch generated this way that I'm tinkering to figure a better way to generate without having to add them to the repo. 

Minor thing in my opinion though 😆 Just the usual thinking too much 😄

Reply:
Ah, yes. With this we now can. Will update